packages/aws-cdk-lib/aws-events/README.md

```diff
@@ -248,3 +248,16 @@ const eventBus = events.EventBus.fromEventBusArn(this, 'ImportedEventBus', 'arn:
 // now you can just call methods on the eventbus
 eventBus.grantPutEventsTo(lambdaFunction);
 ```
+
+## Use a customer managed key
+To use a customer managed key for events on the event bus, use the `kmsKey` attribute.
+
+```ts
+import * as kms from 'aws-cdk-lib/aws-kms';
+
+declare const kmsKey: kms.IKey;
+
+new events.EventBus(this, 'Bus', {
+  kmsKey,
+});
+```
```

Comment:
Might be useful to add an extra note about archives and schema discovery not working with CMK ([docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-eventbus.html#cfn-events-eventbus-kmskeyidentifier)).

Reply:
Thanks.
I added the note.